beetsplug/discogs.py

```diff
@@ -88,6 +91,20 @@ def setup(self, session=None):
         self.discogs_client = Client(USER_AGENT, c_key, c_secret,
                                      token, secret)
 
+    def _time_to_next_request(self):
+        seconds_between_requests = 60 / self.rate_limit_per_minute
+        seconds_since_last_request = time.time() - self.last_request_timestamp
+        seconds_to_wait = seconds_between_requests - seconds_since_last_request
+        if seconds_to_wait > 0:
```

Comment:
There are two positivity checks: one here and one in `wait_for_rate_limiter`. Seems like we only need one of the two.